spec/src/main/asciidoc/microprofile-openapi-spec.adoc

```diff
@@ -135,3 +135,12 @@ overriding any conflicting elements from the current model.
 
 
 == Limitations
+
+=== Internationalization
+The 1.0 version of the MicroProfile OpenAPI spec does not require vendors to
+support multiple languages based on the `Accept-Language`.  One reasonable
+approach is for vendors to support unique keys (instead of hardcoded text) via
+the various <<Documentation Mechanisms>>, so that the implementing framework can
+perform a global replacement of the keys with the language-specific text that
+matches the `Accept-Language` request for the `/openapi` endpoint.  A cache of
+processed languages can be kept to improve performance. 
```

Comment:
This seems reasonable as long as it's consistent with the approach taken by other MP specs.